Makefile

```diff
@@ -1127,13 +1122,11 @@ qmail-log.5
 
 qmail-lspawn: \
 load qmail-lspawn.o spawn.o prot.o slurpclose.o coe.o sig.a wait.a \
-case.a cdb.a fd.a open.a stralloc.a substdio.a error.a str.a \
-fs.a auto_qmail.o uid.o gid.o auto_userp.o auto_userq.o auto_groupn.o \
-auto_spawn.o
+case.a cdb.a fd.a open.a stralloc.a ids.a substdio.a error.a str.a \
+fs.a auto_qmail.o
```

Comment:
`ids.a` should still come after `auto_qmail.o`, and we lost the `auto_spawn.o` dependency